templates/operations.html

```diff
@@ -375,6 +375,11 @@ <h1 id="time-tactic" class="member-title"></h1>
             "source":document.getElementById("queueSource").value,
             "visibility": document.getElementById("queueVisibility").value
         };
+        let planner = document.getElementById("queuePlanner").value;
+        if (planner) {
+        	op["planner"] = planner;
```

Comment:
seems like this should stay the same or set to null and then the backend service should handle no planner being present